### ISSUE TITLE
Text display glitch on Windows and Linux

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#3]
```
When "Include jQuery Mobile ..." or Include "Sencha Touch ..." is selected, the 
"Use phonegap example source ..." line should be extended with "with jQuery 
Mobile" or "with Sencha Touch".  On Windows and Linux, a resize is sometimes 
needed for the "with ..." to be visible.

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 5 Jan 2011 at 6:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 5 Jan 2011 at 7:02
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Actually it's all hosts.  the problem occur on a subsequent wizard invocation 
if the previous invocation ended in a good state with neither of jQuery Mobile 
or Sencha selected.

It seems that Eclipse needs to have enough buffer space already allocated for 
the with strings to show up without a redraw.  Making the else clause in 
validate page add spaces addresses the problem - 

        if (contentsVisible) {
            String withString;
            if (mJqmDialog.jqmChecked()) {
                withString = "with jQuery Mobile";
            } else if (mSenchaDialog.senchaChecked()) {
                withString = "with Sencha Touch";
            } else {
                // needs to be seeded with blanks so that there's space when it needs to appear (Issue 3)
                withString = "                                 ";
            }

The fix will go into 1.1.1
```

Original comment by `paul.beu...@gmail.com` on 10 Jan 2011 at 10:20
- Changed state: **Fixed**
